test/unit/retabled/filter_test.cljc

```diff
@@ -0,0 +1,63 @@
+(ns retabled.filter-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [retabled.filter :as filter]))
+
+(deftest resolve-filter
+  (let [resolve-filter-test (fn [expected column entry]
+                              (is (= expected (filter/resolve-filter column entry))))
+        resolve-filter-test-error (fn [column entry]
+                                    (is (thrown-with-msg? Exception #"Unable to resolve filter" (filter/resolve-filter column entry))))]
+    #_(testing "Correctly handles nil controls and/or entries" ;; Should handle nils
+        (resolve-filter-test nil nil nil)
+        (resolve-filter-test nil nil {:id 1, :name "tom"})
```

Comment:
In this case, keep the `nil` result, but also provide feedback that there is a problem here